hbase-http/src/test/java/org/apache/hadoop/hbase/http/log/TestLogLevel.java

```diff
@@ -84,31 +83,29 @@
   private final static String KEYTAB  = "loglevel.keytab";
 
   private static MiniKdc kdc;
-  private static HBaseCommonTestingUtility htu = new HBaseCommonTestingUtility();
 
   private static final String LOCALHOST = "localhost";
   private static final String clientPrincipal = "client/" + LOCALHOST;
   private static String HTTP_PRINCIPAL = "HTTP/" + LOCALHOST;
-
-  private static final File KEYTAB_FILE = new File(
-      htu.getDataTestDir("keytab").toUri().getPath());
+  private static HBaseCommonTestingUtility htu;
+  private static File keyTabFile;
 
   @BeforeClass
   public static void setUp() throws Exception {
-    BASEDIR = new File(htu.getDataTestDir().toUri().getPath());
-
-    FileUtil.fullyDelete(BASEDIR);
-    if (!BASEDIR.mkdirs()) {
-      throw new Exception("unable to create the base directory for testing");
-    }
     serverConf = new Configuration();
+    htu = new HBaseCommonTestingUtility(serverConf);
```

Comment:
And maybe you want to use the HTU's conf. Do HTU.getConfiguration instead of creating a 'new Configuration()' in line above ... and FYI, for configs that will include hbase config, you want HBaseConfiguration.create.... instead of new Configuration.